layouts/shortcodes/blocks/case-studies.html

```diff
@@ -10,7 +10,7 @@ <h3>{{ $caseStudiesSection.Title }}</h3>
       {{ range $caseStudiesPages }}
       {{ $logo := .Resources.GetMatch "**logo*.png" }}
       <div>
-        {{ with $logo }}<img src="{{ .RelPermalink }}">{{ end }}
+        {{ with $logo }}<img src="{{ .RelPermalink }}" alt="{{ .Title }}">{{ end }}
         <p>"{{ .Params.quote | truncate 100 }}"</p>
         <a href="{{ .RelPermalink }}">{{ T "main_read_more" }}</a>
       </div>
```

Comment:
As per https://webaim.org/techniques/alttext/#logos the logos https://kubernetes.io/case-studies/ on should be updated

Reply:
> As per https://webaim.org/techniques/alttext/#logos the logos https://kubernetes.io/case-studies/ on should be updated


@Rajakavitha1 I guess you are mentioning the redundant use of the ``logo`` keyword in the alt text.

If it is, the ``logo`` field associated with all the case studies has to be changed in order to reflect that.

Here's an [example](https://github.com/kubernetes/website/blob/master/content/en/case-studies/chinaunicom/index.html#L8). This will be a big change. We can include this in a separate PR.

Reply:
Yes, the _ , the word logo and the .png . Please feel free to raise a new [WIP] PR or an issue for that so that we can track it.

Reply:
Here's the [PR](https://github.com/kubernetes/website/pull/16691)